src/Native/System.Net.Http.Native/pal_easy.cpp

```diff
@@ -256,6 +267,31 @@ extern "C" int32_t HttpNative_RegisterSslCtxCallback(CURL* curl,
     return curl_easy_setopt(curl, CURLOPT_SSL_CTX_FUNCTION, &ssl_ctx_callback);
 }
 
+static int debug_callback(CURL* curl, curl_infotype type, char *data, size_t size, void *userPointer)
```

Comment:
nit: * spacing.

Which reminds me: can you run the format-code.py script on the new code? You have to modify it to run it - remove the sys.exit(0) line, and uncomment the rest of the file. I'm not sure why it all got commented out, or if it can be reverted.